roles/rsyslog/roles/input_roles/journal/defaults/main.yaml

```diff
@@ -0,0 +1,116 @@
+---
+# journal configuration
+# ---------------------
+
+# journal rpm packages
+# adding rsyslog_logging_packages
+# ------------------
+
+# .. envvar:: rsyslog_logging_packages
+#
+# List of rpm packages for Common Logging.
+rsyslog_journal_packages: ['rsyslog-mmnormalize', 'rsyslog-mmjsonparse', 'libfastjson', 'liblognorm', 'libestr']
```

Comment:
@richm Do we need all these packages?

Reply:
Yes, if you are doing processing for output to viaq/openshift common logging.  They are not necessarily needed for other processing, although anything except trivial parsing will require rsyslog-mmnormalize, and the `lib` packages are dependencies of rsyslog-mmnormalize

Reply:
rsyslog-mmjsonparse is needed primarily for docker json-file logs, or if you have nested JSON in your log/message/MESSAGE field you want to parse (although rsyslog-mmnormalize can also parse JSON)

Reply:
@pcahyna @richm In that case, I think we should keep all of them. Do you agree?
Are they available on the RHEL channels? Do we need to mention anything in the documentation in order for the users to be able to consume them?